salt/sdb/redis_sdb.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+'''
+Redis SDB module
+================
+
+This module allows access to Redis  using an ``sdb://`` URI.
+
+Like all SDB modules, the Redis module requires a configuration profile to
+be configured in either the minion or master configuration file. This profile
+requires very little. For example:
+
+.. code-block:: yaml
+
+    sdb_redis:
+      driver: redis
+      host: 127.0.0.1
+      port: 6379
+      password: pass
+      db: 1
+
+The ``driver`` refers to the Redis module, all other options are optional.
+For option details see: https://redis-py.readthedocs.io/en/latest/.
+
+'''
+from __future__ import absolute_import
```

Comment:
We also need to import `print_function` and `unicode_literals` here for PY3 compatibility.

Reply:
Good points - added: https://github.com/saltstack/salt/pull/48239/commits/30cd544d7ebc2491db6034c4215d59b73db58def

Regarding `print_function` and `unicode_literals`: I've seen this has been added across all the modules, and I'm afraid I didn't understand that - could you please explain why is `print_function` (for example) needed although the module doesn't call the `print()` function? Thanks!